### ISSUE TITLE
[New term] `eBPF`

### ISSUE THREAD

**Issue by ams0** [cncf/glossary#1448]
#### What this term is about? (briefly)

eBPF (extended Berkley Packet Filter) is a central component of many CNIs and cloud native tools (Pixie, Inspector Gadget and more).
 
#### Why this term needs to be added? (briefly) 

The eBPF technology is quickly becoming central to cloud native architectures and needs to be explained in layman terms to the non-initiated to the arcane world of Linux kernel networking.

### 2. Checklist

- [X] The suggested term fits the [cloud native definition](https://github.com/cncf/toc/blob/main/DEFINITION.md) or is essential to understanding cloud native terms. (We hope to avoid adding general or broad terminologies)
- [X] There is no existing issue yet. [(shift+click to check existing issues)](https://github.com/cncf/glossary/issues?q=label%3A%22new+term%22)
- [X] The term has not already been declined in the past. [(shift+click to check)](https://github.com/cncf/glossary/issues?q=is%3Aissue+is%3Aclosed+label%3A%22triage%2Fnot+accepted%22)
- [X] The term does not already exist in the [glossary](https://glossary.cncf.io).
- [X] I understand that maintainers will assess if the term should be part of the Glossary. (Please be patient)
- [X] I want to work on this term. (If not checked, maintainers will assign another volunteer)

### (Note)

Maintainers will assign a label after assessment procedure. 
(from `triage/awaiting` to `triage/accepted` or `triage/not accepted`)

A contributor can be assigned to this issue after the issue get the `triage/accepted` label.
The assignee needs to check [Style Guide](https://glossary.cncf.io/style-guide/).

**Comment by seokho-son** [comment]
Hi @ams0 
Thank you for the suggestion. Maintainers agree with your suggestion ! :)

Although the Cloud Native Glossary project tend to avoid including too complex technical term, we also think eBPF needs to be included in the Glossary considering importance of it.
Please try to explain eBPF easily so that non-technical folks can understand it easily.

Please check following resources before you start. :)
- [Style Guide](https://glossary.cncf.io/style-guide/)
- [How To Contribute](https://glossary.cncf.io/contribute/)

(CC @iamNoah1 @CathPag @jihoon-seo @JasonMorgan )

**Comment by CathPag** [comment]
Hi @ams0, nice to see you here :) Are you still interested in working on this? It's been two months since this issue was assigned to you, and there is someone else interested in tackling it.

**Comment by iamNoah1** [comment]
friendly ping @ams0

**Comment by MaryamTavakkoli** [comment]
I would like to work on this term.

**Comment by CathPag** [comment]
Sure, here you go, @MaryamTavakkoli. Please be sure to read the [style guide](https://glossary.cncf.io/style-guide/) before getting started.
